content/kubelb/main/support-policy/_index.en.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "Support Policy"
+date = 2024-03-15T00:00:00+01:00
+weight = 40
++++
+
+KubeLB has an open-source community edition and an enterprise edition. The community edition is free to use and is licensed under the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0).
+
+The enterprise edition, backed by an active subscription for [KubeLB](https://www.kubermatic.com/products/kubelb/), provides comprehensive enterprise-grade support, guaranteed SLAs, priority issue resolution, and direct access to our expert engineering team. Our enterprise customers benefit from personalized technical guidance, architectural consulting, and our commitment to their production success.
+
+## Enterprise Edition Support
+
+As a default, our support covers the following:
+
+- Bug reports
+- Feature requests
+- Documentation issues
+- Debugging for issues related to KubeLB
+
```

Comment:
I'd be more cautious here, and take out Feature requests, because this could cause confusion, and one might think that we'll just implement any requests if they are our customers, and gain false expectations.

Like:

 - Debugging for issues related to KubeLB
 - Enhancing documentation
 - Fixing bugs that block the usage of the platform

Reply:
Updated